packages/cosmo-dycore/package.py

```diff
@@ -182,16 +162,14 @@ def cmake_args(self):
     @run_after('install')
     @on_package_attributes(run_tests=True)
     def test(self):
-        cwd = os.path.join(self.root_cmakelists_dir, 'test/tools')
         if '+build_tests' in self.spec:
-            with open(os.path.join(cwd, 'spec.yaml'), mode='w') as f:
-                f.write(self.spec.to_yaml())
             try:
                 subprocess.run([
-                    './test_dycore.py', '-s', 'spec.yaml', '-b',
+                    './test_dycore.py', '-s',
```

Comment:
the command installcosmo provdides the custom option `--test=dycore`, it will tests the dycore plus cosmo.
Did you check if this works w/o the patches as well?

Reply:
I was not aware that this exists.
Can this be tested right now, I thought cosmo +dycore cannot be testet atm?

Reply:
Ah I think it works as long as you don't pass +serialize to the cosmo spec.
This does not work.